core/commands/configure_network_command.rb

```diff
@@ -0,0 +1,116 @@
+# frozen_string_literal: true
+
+require_relative '../models/network_config_file'
+require_relative '../services/machine_configurator'
+require 'net/ssh'
+
+# This class loads ssh keys to configuration or selected nodes.
+class ConfigureNetworkCommand
```

Comment:
The class MUST be the subclass of the BaseCommand and perform the same initialization steps as other Commands do.